### PR TITLE
Feature: Add agent tooltip in summary page

### PR DESCRIPTION
**Pull request by Bilelkihal**
### Done in this PR:
I added a tooltip when we hover an agent in people and organizations bloc in summary page,
The information I'm displaying is: **Type (using an icon)**, **name**, **email**, **identifiers** and **affiliations** of the agent.
there are two types of agents:
- person:
image

- Organization:
image

** Also I added an icon to indicate the type of the agent in the agent chip

Ps. there are some agents have no type (only name and email) and they are considered as persons.

### CODE REVIEW

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
See requested changes in the code,
and for the UI, make it more smaller, as the marckup bellow 
image

File: app/assets/images/icons/person.svg
[the remark this answers is not in the document]
Comment (Bilelkihal) [reply app/assets/images/icons/person.svg:1]: People icon is different:
image

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Can you also update the chips content to be like this 
image

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
I need the fix of word breaking and remove the scroll before merging this

**Comment by Bilelkihal** [comment]
@syphax-bouazzouni 
I removed the scroll indicators, now for the text overflow we do text-truncate, 
Concerning the word break I saw the license PR and the change I did was just for the license badge.

**Pull request merged**